src/main/java/htsjdk/samtools/SAMTagUtil.java

```diff
@@ -41,82 +43,141 @@
+    /** @deprecated use {@link SAMTag#getBinaryTag()} instead. */
+    @Deprecated public final short MD = SAMTag.MD.getBinaryTag();
+    /** @deprecated use {@link SAMTag#getBinaryTag()} instead. */
+    @Deprecated public final short MQ = SAMTag.MQ.getBinaryTag();
+    /** @deprecated use {@link SAMTag#getBinaryTag()} instead. */
+    @Deprecated public final short NH = SAMTag.NH.getBinaryTag();
+    /** @deprecated use {@link SAMTag#getBinaryTag()} instead. */
+    @Deprecated public final short NM = SAMTag.NM.getBinaryTag();
+    /** @deprecated use {@link SAMTag#getBinaryTag()} instead. */
+    @Deprecated public final short OQ = SAMTag.OQ.getBinaryTag();
+    /** @deprecated use {@link SAMTag#getBinaryTag()} instead. */
+    @Deprecated public final short OP = SAMTag.OP.getBinaryTag();
+    /** @deprecated use {@link SAMTag#getBinaryTag()} instead. */
+    @Deprecated public final short OC = SAMTag.OC.getBinaryTag();
+    /** @deprecated use {@link SAMTag#getBinaryTag()} instead. */
+    @Deprecated public final short OF = SAMTag.OF.getBinaryTag();
+    /** @deprecated use {@link SAMTag#getBinaryTag()} instead. */
+    @Deprecated public final short OR = SAMTag.OR.getBinaryTag();
+    /** @deprecated use {@link SAMTag#getBinaryTag()} instead. */
+    @Deprecated public final short OX = SAMTag.OX.getBinaryTag();
+    /** @deprecated use {@link SAMTag#getBinaryTag()} instead. */
+    @Deprecated public final short PG = SAMTag.PG.getBinaryTag();
+    /** @deprecated use {@link SAMTag#getBinaryTag()} instead. */
+    @Deprecated public final short PQ = SAMTag.PQ.getBinaryTag();
+    /** @deprecated use {@link SAMTag#getBinaryTag()} instead. */
+    @Deprecated public final short PT = SAMTag.PT.getBinaryTag();
+    /** @deprecated use {@link SAMTag#getBinaryTag()} instead. */
+    @Deprecated public final short PU = SAMTag.PU.getBinaryTag();
+    /** @deprecated use {@link SAMTag#getBinaryTag()} instead. */
+    @Deprecated public final short QT = SAMTag.QT.getBinaryTag();
+    /** @deprecated use {@link SAMTag#getBinaryTag()} instead. */
+    @Deprecated public final short Q2 = SAMTag.Q2.getBinaryTag();
+    /** @deprecated use {@link SAMTag#getBinaryTag()} instead. */
+    @Deprecated public final short QX = SAMTag.QX.getBinaryTag();
+    /** @deprecated use {@link SAMTag#getBinaryTag()} instead. */
+    @Deprecated public final short R2 = SAMTag.R2.getBinaryTag();
+    /** @deprecated use {@link SAMTag#getBinaryTag()} instead. */
+    @Deprecated public final short RG = SAMTag.RG.getBinaryTag();
+
     /** @deprecated use BC instead */
     @Deprecated
-    public static final short RT = SAMTag.RT.getBinaryTag();
-    public static final short RX = SAMTag.RX.getBinaryTag();
+    public final short RT = SAMTag.RT.getBinaryTag();
+
+    /** @deprecated use {@link SAMTag#getBinaryTag()} instead. */
+    @Deprecated public final short RX = SAMTag.RX.getBinaryTag();
     /** @deprecated reserved tag for backwards compatibility only */
+
```

Comment:
Stray whitespace separating this comment from the thing it's commenting on

Reply:
gone